unifi/Dockerfile

```diff
@@ -8,13 +8,13 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # Setup base system
 RUN \
     apt-get update \
+    && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
-        binutils=2.34-6ubuntu1.7 \
-        jsvc=1.0.15-8 \
-        libcap2=1:2.32-1 \
-        logrotate=3.14.0-4ubuntu3 \
+        binutils=2.34-* \
+        libcap2=1:2.32-* \
+        logrotate=3.14.* \
```

Comment:
Please restore exact version pinning. We have an bot upgrading them.
Also, this messes with Docker caching.

Reply:
Ah ok, was not aware of the bot, and saw some build issues related to that in another PR. I'll drop it.

Reply:
NB: current `main` is missing this update:
```
diff --git a/unifi/Dockerfile b/unifi/Dockerfile
index 9398a12..9aef07d 100755
--- a/unifi/Dockerfile
+++ b/unifi/Dockerfile
@@ -10,7 +10,7 @@ RUN \
     apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
-        binutils=2.34-6ubuntu1.7 \
+        binutils=2.34-6ubuntu1.8 \
         libcap2=1:2.32-1 \
         logrotate=3.14.0-4ubuntu3 \
         mongodb-server=1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 \
```

Reply:
Dropped this commit from the PR.